src/chia_log/log_consumer.py

```diff
@@ -50,53 +54,69 @@ def _notify_subscribers(self, logs: str):
 
 
 class FileLogConsumer(LogConsumer):
-    """Specific implementation for a simple file consumer"""
-
     def __init__(self, log_path: Path):
-        logging.info("Enabled file log consumer.")
         super().__init__()
         self._log_path = log_path
+        self._expanded_log_path = ""
         self._is_running = True
         self._thread = Thread(target=self._consume_loop)
         self._thread.start()
+        self._log_size = 0
 
     def stop(self):
         logging.info("Stopping")
         self._is_running = False
 
+    @retry((FileNotFoundError, PermissionError), delay=2)
     def _consume_loop(self):
-        expanded_user_log_path = str(self._log_path.expanduser())
-        logging.info(f"Consuming log file from {expanded_user_log_path}")
+        while self._is_running:
+            for log_line in Pygtail(self._expanded_log_path, read_from_end=True, paranoid=True):
```

Comment:
Do we know if `paranoid=True` is really necessary? I think it might add quite a lot of unnecessary disk writes.

Reply:
I agree, I've tested both and `paranoid` seemed more reliable at picking up errors, but having again retested without paranoid for a bit just now, it seems to work equally reliable, so it might have just been something else that changed while I was busy testing multiple config parameters.

I would suggest to leave it out for now and put it back in if we see any more issues